include/tss2/tss2_tcti_spi.h

```diff
@@ -0,0 +1,45 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2020 Fraunhofer SIT. All rights reserved.
+ */
+#ifndef TSS2_TCTI_SPI_H
+#define TSS2_TCTI_SPI_H
+
+#include "tss2_tcti.h"
+#include <driver/spi_master.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define TSS2_TCTI_SPI_PLATFORM_MAGIC 0x61F7D74109DB0D8BULL
+
+typedef struct TSS2_TCTI_SPI_PLATFORM TSS2_TCTI_SPI_PLATFORM;
+typedef void (*TSS2_TCTI_SPI_PLATFORM_SLEEP_MS_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform, int milliseconds);
+typedef void (*TSS2_TCTI_SPI_PLATFORM_START_TIMEOUT_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform, int milliseconds);
+typedef bool (*TSS2_TCTI_SPI_PLATFORM_TIMEOUT_EXPIRED_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform);
+typedef void (*TSS2_TCTI_SPI_PLATFORM_SPI_ACQUIRE_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform);
+typedef void (*TSS2_TCTI_SPI_PLATFORM_SPI_RELEASE_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform);
+typedef TSS2_RC (*TSS2_TCTI_SPI_PLATFORM_SPI_TRANSFER_FUNC) (TSS2_TCTI_SPI_PLATFORM* platform, const void *data_out, void *data_in, size_t cnt);
+
+struct TSS2_TCTI_SPI_PLATFORM {
+    uint64_t magic;
```

Comment:
This should not be necessary, since we have type safety whenever this struct is used.

Reply:
Magic has been removed from TSS2_TCTI_SPI_PLATFORM.